data/completion/etelpmoc.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+# -*- bash -*-
```

Comment:
does adding this keep shellcheck happy?

Reply:
Hmm, no, not really.

Bummer. Shellcheck wants a shebang, rpmlint doesn't want one unless the script is executable.

Maybe the (silly) solution is to make both executable?

Reply:
With both executable and explicitly using `#!/bin/bash` we get this:
```
zyga@tumbleweed:~/snapd> shellcheck data/completion/etelpmoc.sh 

In data/completion/etelpmoc.sh line 90:
        case "$opt" in
        ^-- SC2220: Invalid flags are not handled. Add a *) case.


In data/completion/etelpmoc.sh line 212:
printf "%s\n" "${COMPREPLY[@]}"
          ^-- SC1117: Backslash is literal in "\n". Prefer explicit escaping: "\\n".

zyga@tumbleweed:~/snapd> shellcheck data/completion/
complete.sh  etelpmoc.sh  snap         
zyga@tumbleweed:~/snapd> shellcheck data/completion/complete.sh 

In data/completion/complete.sh line 83:
            COMPREPLY=( $( command grep -v '[[:cntrl:];&?*{}]' ) )
                        ^-- SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).


In data/completion/complete.sh line 119:
        if [[ "$(which "$1")" =~ /snap/bin/ && ( -e "/var/lib/snapd/$etel" || -e "/$etel" ) ]]; then
                 ^-- SC2230: which is non-standard. Use builtin 'command -v' instead.

```

Reply:
I think I prefer this

Reply:
I pushed the opposite approach now.